content/en/blog/2019/introducing-istio-operator/index.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Introducing the Istio Operator
+description: Introduction to Istio's new operator-based installation and control plane management feature.
+publishdate: 2019-11-04
+subtitle:
+attribution: Martin Ostrowski (Google), Frank Budinsky (IBM)
+keywords: [install,configuration,istioctl,operator]
+target_release: 1.4
+---
+
+Kubernetes [operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) provide
+a pattern for encoding human operations knowledge in software and are a popular way to simplify
+the administration of software infrastructure components.
+Because Istio is complex to administer, it's a natural candidate for an operator.
+
+Up until now, [Helm](https://github.com/helm/helm) has been the primary tool to install and upgrade Istio.
+In addition to needing to install another tool, this approach is not so desireable because it lacks
+schematic validation, allowing invalid configurations to be applied, which can then only be detected
+after installation. With operators, on the other hand, the API is schematized in a
+CRD (Custom Resource Definition) which is validated.
```

Comment:
Helm users will take exception to this since it's in principle possible to validate the schema (though not conveniently). There are other reasons for getting rid of helm which are explained in the design doc but for the blog I suggest we leave this section out and not say anything negative about Helm.

Reply:
We can tone it down to sound less negative about Helm, but I think it's very important that this blog explains why we're replacing it. It's a significant disruption for users, so we need to be clear about why the new approach is better.

I guess another benefit of operator is the easy continuous incremental update provided by the controller. Can you point me to the where in the design doc reasons are explained? I'll take another pass at justifying the change more diplomatically.

Reply:
How about dropping the above paragraph and adding something along the lines of the following, after the next paragraph:
"The new istioctl CLI and operator controller both validate configuration schemas and perform a range of checks for installation change or upgrade. These checks are tightly integrated with the tools to prevent common errors and simplify troubleshooting."
The fact is we did have a lot of problems with Helm, but I'd rather not dwell on it. Maybe we can say something more vague like "increased robustness"...